### DISCUSSION TITLE
Battlesnake image export endpoint

### DISCUSSION THREAD

**Discussion by chris-bsnake** [BattlesnakeOfficial/feedback#42] [Feature Requests]
Create a new API endpoint for the Exporter that can render a simple image of a single Battlesnake given a set of attributes like head, tail, color and size. This would be used in all kinds of tools and on the play.battlesnake.com site to replace the snake graphics built via CSS.

**Requirements:**

- GET request:   /snake/png?head=x&tail=y&color=ff0000&width=100&height=20
- Generates a PNG of the snake on a transparent background with no padding
- Takes 6 parameters:
-- head (string): the ID of the head customization to use (i.e. pixel) 
-- tail (string): the ID of the tail customization to use (i.e. sharp) 
-- color (string): The hex code of the color to render the snake in
-- width (number): The width of the generated image, in pixels
-- height (number): The height of the generated image, in pixels

<img width="198" alt="exported snake example" src="https://user-images.githubusercontent.com/53871533/108743427-a046d580-74ed-11eb-9c37-fad6a3389f80.png">

**Comment by bvanvugt** [comment]
Created an issue (#71) to track this.
